client/src/pages/Profile.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useState, useEffect } from 'react';
+import { Container, Row, Button, Col, Card } from 'react-bootstrap';
+import SolutionCard from '../components/SolutionCard';
+import { User } from '../../../server/models/User';
+import "./Profile.css";
+
+const Profile = () => {
+const sampleUser = {
+  username: 'Alexander Apostolu',
+  preferredName: 'Alex',
+  email: 'apostolu240@gmail.com',
+  score: 100
+};
+
+const sampleSolutions = [
+  {
+    id: 1,
+    title: "Solution 1",
+    description: "Description of Solution 1",
+    imgSrc: "https://example.com/image1.jpg",
+    difficulty: "medium"
+  },
+  {
+    id: 2,
+    title: "Solution 2",
+    description: "Description of Solution 2",
+    imgSrc: "https://example.com/image2.jpg",
+    difficulty: "hard"
+  },
+];
+
+const sampleComments = [
+  {
+    name: "Factory Design Pattern",
+    content: "This solution is amazing!",
+    createdAt: "Wednesday",
+  },
+  {
+    name: "Builder Design Pattern",
+    content: "jobizdan",
+    createdAt: "Thursday",
+  }
+];
+
+const [user, setUser] = useState(sampleUser);
+const [solutions, setSolutions] = useState(sampleSolutions);
+const [comments, setComments] = useState(sampleComments);
```

Comment:
Ideally we want to define the samples in the backend and return those so our frontend logic is complete and not hardcoded. And then we can implement the backend logic

Let me know if you need help with that

Reply:
Done!